dev-haskell/integer-conversion/integer-conversion-0.1.1-r1.ebuild

```diff
@@ -17,7 +17,7 @@ LICENSE="BSD"
 SLOT="0/${PV}"
 KEYWORDS="~amd64"
 
-RDEPEND=">=dev-haskell/primitive-0.9.0.0:=[profile?] <dev-haskell/primitive-0.10:=[profile?]
+RDEPEND=">=dev-haskell/primitive-0.9.0.0:=[profile?]
```

Comment:
Another removal of an upper bound with no change to `CABAL_HACKAGE_REVISION` or `CABAL_CHDEPS`.